ethcore/src/ethereum/ethash.rs

```diff
@@ -222,6 +222,8 @@ impl Engine<EthereumMachine> for Arc<Ethash> {
 
 	fn maximum_uncle_count(&self, _block: BlockNumber) -> usize { 2 }
 
+	fn maximum_gas_limit(&self) -> Option<U256> { Some(0x7fffffffffffffffu64.into()) }
```

Comment:
Can you use `underscores` for this number to make it a little bit more readable?